src/scene/BattleScene.ts

```diff
@@ -52,32 +52,36 @@ export class BattleScene extends Scene {
 
     this.creatureSpawner
       .setScene(this)
-      .apply(this.getHeight() * 0.6, this.getHeight() * 0.22);
+      .apply(this.ground.getGroundY());
 
     this.player.setAnimation(new PlayerAnimation(this.imageRegistry, this.collisionRegistry));
+    this.player.setScale(1.25);
     this.player.setX(this.getWidth() / 9);
-    this.player.setY(this.getHeight() * 0.61);
+    this.player.setY(this.ground.getGroundY() - (this.player.getCollision().getHeight() + this.player.getCollision().getTop()) * this.player.getScale());
```

Comment:
Maybe we should create `Entity#setOnGround`

Reply:
I will👌